### PR TITLE
Add Supporting Windows Docker Container Hosted API Services

### PR DESCRIPTION
**Pull request by croos-msft**
This PR Adds a DockerFile and Run Script for a Windows DataContainer that will hold the contents of the repo. It also adds a parameter setting to the AzurePipelines yaml file that adds windows and the windows dockerfile as run parameters

### CODE REVIEW

**Comment by raych1** [comment]
Adding the right reviewer from rpaas team.

@croos-msft , I would like to know more about your scenario. Feel free to ping me in IM.

**Review by leal-alvaro — APPROVED** [review]
I believe there's no impact on RPaaS, since the linux-based data container our service uses will still be generated. Please test before merging.

**Comment by raych1** [comment]
@croos-msft , do you push the image to RpaaS ACR? @leal-alvaro, are you ok with that?

**Comment by leal-alvaro** [comment]
> @croos-msft , do you push the image to RpaaS ACR? @leal-alvaro, are you ok with that?

It would be better for this to be consumed by a separate pipeline, and push the data container to another ACR.

File: .azure-pipelines/azure-pipelines-data-container.yml [hunk 1]
```diff
@@ -29,13 +47,13 @@ steps:
...
-    targetFolder: $(Build.SourcesDirectory)/scripts/datacontainer
+    targetFolder: $(Build.SourcesDirectory)/${{ parameters.PathToDockerfile }}
 
 - task: Docker@2
   displayName: Build and Push
   inputs:
     command: buildAndPush
-    Dockerfile: scripts/datacontainer/Dockerfile
+    Dockerfile: ${{ parameters.PathToDockerfile }}/Dockerfile
     repository: $(CONTAINER_REGISTRY_REPOSITORY_NAME)
```
Comment (weshaggard) [inline .azure-pipelines/azure-pipelines-data-container.yml:57]: This variable is currently defined in the pipeline variables but if we want to publish a unique windows and ubuntu image we will need to give them different names. We also need confirmation about which container registry to publish this new one to before we go and try running this.

**Pull request merged**